content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,702 @@
+
+        - :guilabel:`Default filter`: when saving a search, check this box to make it the default
+          search filter when the :guilabel:`Pipeline Analysis` page is opened.
+        - :guilabel:`Shared`: when saving a search, check this box to make it available to other
+          users.
+
+.. _win_loss/custom_filters:
+
+Add custom filters and groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to the pre-made options in the search bar, the :guilabel:`Pipeline Analysis` page can
+also utilize custom filters and groups.
+
+Custom filters are complex rules that further customize the search results, while custom groups
+display the information in a more organized fashion.
+
+**To add a custom filter:**
+
+1. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` icon next to the
+   search bar.
+2. In the drop-down menu, click :guilabel:`Add Custom Filter`.
+3. The :guilabel:`Add Custom Filter` pop-up window appears with a default rule (:guilabel:`Country
+   is in _____`) comprised of three unique fields. These fields can be edited to make a custom rule,
+   and multiple rules can be added to a single custom filter.
+4. To edit a rule, start by clicking the first field (:guilabel:`Country`), and select an option
+   from the drop-down menu. The first field determines the primary subject of the rule.
+
+.. image:: win_loss/custom-filter-first-field.png
+   :align: center
+   :alt: The first field of a custom filter determines the primary subject of the rule.
+
+5. Next, click the second field, and select an option from the drop-down menu. The second field
+   determines the relationship of the first and third fields, and is usually an "is" or "is not"
+   statement, but can be "greater than/less than" statements, whether the first field "contains"
+   certain elements, and more.
+
+.. image:: win_loss/custom-filter-second-field.png
+   :align: center
+   :alt: The second field of a custom filter determines the relationship of the subjects.
+
+6. Finally, click the third field, and select an option from the drop-down menu. The third field
+   determines the secondary subject of the rule.
+
+.. image:: win_loss/custom-filter-third-field.png
+   :align: center
+   :alt: The third field of a custom filter determines the secondary subject of the rule.
+
+7. With all three fields selected, the rule is complete.
+
+   - **To add more rules:** click :guilabel:`New Rule` and repeat steps 4-7, as needed.
+   - **To delete a rule:** click the :guilabel:`🗑️ (trash)` icon to the right of the rule.
+   - **To duplicate an existing rule:** click the :guilabel:`➕ (plus sign)` icon to the right of the
+     rule.
+   - **To create more complex rules:** click the :guilabel:`Add branch` icon to the right of the
+     rule. This adds another modifier below the rule for adding an "all of" or "any of" statement.
+
+.. image:: win_loss/custom-filter-add-branch.png
```

Comment:
show a much more completed example to visually drive in the lesson and the value